src/Sentry.Maui/Internal/MauiGestureRecognizerEventsBinder.cs

```diff
@@ -0,0 +1,197 @@
+                    drag.DropCompleted += OnDropCompletedGesture;
+                }
+                break;
+
+            case PanGestureRecognizer pan:
+                pan.PanUpdated -= OnPanGesture;
+
+                if (bind)
+                {
+                    pan.PanUpdated += OnPanGesture;
+                }
+                break;
+
+            case PointerGestureRecognizer pointer:
+                pointer.PointerEntered -= OnPointerEnteredGesture;
+                pointer.PointerExited -= OnPointerExitedGesture;
+                pointer.PointerMoved -= OnPointerMovedGesture;
+                pointer.PointerPressed -= OnPointerPressedGesture;
+                pointer.PointerReleased -= OnPointerReleasedGesture;
+
+                if (bind)
+                {
+                    pointer.PointerEntered += OnPointerEnteredGesture;
+                    pointer.PointerExited += OnPointerExitedGesture;
+                    pointer.PointerMoved += OnPointerMovedGesture;
+                    pointer.PointerPressed += OnPointerPressedGesture;
+                    pointer.PointerReleased += OnPointerReleasedGesture;
+                }
+                break;
+        }
+    }
+
+    private static void OnPointerReleasedGesture(object? sender, PointerEventArgs e) => _addBreadcrumb?.Invoke(new(
+        sender,
+        nameof(PointerGestureRecognizer.PointerReleased),
+        ToPointerData(e)
+    ));
+
+    private static void OnPointerPressedGesture(object? sender, PointerEventArgs e) => _addBreadcrumb?.Invoke(new(
+        sender,
+        nameof(PointerGestureRecognizer.PointerPressed),
+        ToPointerData(e)
+    ));
+
+    private static void OnPointerMovedGesture(object? sender, PointerEventArgs e) => _addBreadcrumb?.Invoke(new(
+        sender,
+        nameof(PointerGestureRecognizer.PointerMoved),
+        ToPointerData(e)
+    ));
+
+    private static void OnPointerExitedGesture(object? sender, PointerEventArgs e) => _addBreadcrumb?.Invoke(new(
+        sender,
+        nameof(PointerGestureRecognizer.PointerExited),
+        ToPointerData(e)
+    ));
+
+    private static void OnPointerEnteredGesture(object? sender, PointerEventArgs e) => _addBreadcrumb?.Invoke(new(
+        sender,
+        nameof(PointerGestureRecognizer.PointerEntered),
+        ToPointerData(e)
+    ));
+
+    private static IEnumerable<(string Key, string Value)> ToPointerData(PointerEventArgs e) =>
+    [
+        // some of the data here may have some challenges being pulled out
+        #if ANDROID
+        ("MotionEventAction", e.PlatformArgs?.MotionEvent.Action.ToString() ?? string.Empty)
+        //("MotionEventActionButton", e.PlatformArgs?.MotionEvent.ActionButton.ToString() ?? String.Empty)
+        #elif IOS
+        ("State", e.PlatformArgs?.GestureRecognizer.State.ToString() ?? string.Empty),
+        //("ButtonMask", e.PlatformArgs?.GestureRecognizer.ButtonMask.ToString() ?? String.Empty)
```

Comment:
Again, we should probably remove this... don't want loads of commented out code in the solution.

Reply:
Done